test/integration/partials-test.js

```diff
@@ -0,0 +1,50 @@
+const assert = require('assert');
+const path = require('path');
+const dist = path.resolve(__dirname, '..', '..', 'dist');
+
+describe('Partials test', function() {
+  it('Partials include RSS urls', async function() {
+    const expected = '/feed.xml';
+    const partialPath = path.join(dist, 'en', 'index.json');
+    const partial = require(partialPath);
+
+    assert.ok(
```

Comment:
If you use `assert.deepStrictEqual(actual, expected)` then you don't need to write your own message because it will log the difference between the actual and expected.